File: sdi3-14.Cli-REST/src/uo/sdi/clients/rest/menu/items/LogoutMenuItem.java
package uo.sdi.clients.rest.menu.items;

import java.util.ArrayList;
import java.util.List;

import uo.sdi.clients.rest.menu.MenuItem;

public class LogoutMenuItem implements MenuItem {

	@Override
	public String getMenuName() {
		return "Cerrar sesion";
	}

	@Override
	public List<MenuItem> execute() {
		List<MenuItem> items = new ArrayList<MenuItem>();

		items.add(new LoginMenuItem());
		return items;
	}

}

File: sdi3-14.Cli-REST-CSharp/sdi3-14.Cli-REST-CSharp/menu/menuItems/LogoutMenuItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace sdi3_14.Cli_REST_CSharp.menu.menuItems
{
    class LogoutMenuItem : MenuItem
    {
        public IList<MenuItem> Execute()
        {
            IList<MenuItem> items = new List<MenuItem>();

            items.Add(new LoginMenuItem());

            return items;
        }

        public string GetMenuName()
        {
            return "Cerrar sesion";
        }
    }
}
